newrelic/resource_newrelic_alert_condition.go

```diff
@@ -136,8 +136,9 @@ func resourceNewRelicAlertCondition() *schema.Resource {
 			"violation_close_timer": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				ValidateFunc: intInSlice([]int{1, 2, 4, 8, 12, 24}),
-				Description:  "Automatically close instance-based incidents, including JVM health metric incidents, after the number of hours specified. Must be: 1, 2, 4, 8, 12 or 24.",
+				ValidateFunc: validation.IntBetween(1, 720),
+				Default:      72,
+				Description:  "Automatically close instance-based incidents, including JVM health metric incidents, after the number of hours specified. Must be between 1 and 720 hours. Defaults to 72.",
```

Comment:
@founddrama a small question - does the API now expect any value between 1 and 720 and not distinct values such as 1, 2, 4, 8, 12, 24 (as it previously did - or am I mistaken in thinking this was only a validation done in Terraform and not in the API)?

I just wanted to post this question, as I see the UI wouldn't have support to have any random value as `violation_close_timer` - would like to hear your thoughts on this 😄 

<img width="391" alt="image" src="https://github.com/newrelic/terraform-provider-newrelic/assets/127438038/e861106e-e88b-49d1-9949-75a474abe03e">

Reply:
@pranav-new-relic The Terraform provider is performing validation above and beyond what's taking place in the underlying API. Ultimately, that's the source of this regression to begin with.

The change here will accept _any integer between 1 and 720_, representing the `violation_close_timer` as a value _in hours_.

`violation_close_timer` is mainly kept around for backward compatibility. Its previous allowable values (i.e., 1, 2, 4, 8, 12, 24) are a subset of the current valid rules. If you examine these conditions in the UI, you'll see that it accepts values as low as 5 minutes. However, to add support for minutes in Terraform for this condition type would be a breaking change for this field. The change in this PR allows existing conditions (i.e., those using one of the six previously restricted valid values) to continue working.

**As a side note:** the screenshot you're including appears to be from the NRQL condition UI and _not_ from the APM condition UI. NRQL conditions are configured using a different Terraform resource (i.e., `nrql_alert_condition`) which does not expose the `violation_close_timer` and instead uses `violation_time_limit_seconds`.

Reply:
Oops, so sorry - I realised just now that was out of an NRQL Alert Condition. Thank you for clarifying, and the detailed explanation, @founddrama - really appreciate it 😄 I'll get this merged soon.